packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -3498,6 +3498,10 @@ describe('ComboBox', function () {
         triggerPress(clearButton);
       });
 
+      act(() => {
```

Comment:
?

Reply:
I assume it was related to the list layout bugfix